### ISSUE TITLE
Indicate what toolkit each app uses

### ISSUE THREAD

**Issue by benwaffle** [LewisVo/Awesome-Linux-Software#209]
I'm sure users would like to know if an app is written using GTK, Qt, Electron, Java, ...

**Comment by purachina** [comment]
I totally agree, I think it is necessary to indicate what kind of toolkit every software uses in order to make the choice easier.
Probably a favicon for each toolkit.

**Comment by LewisVo** [comment]
@purachina  @benwaffle thank you guys for sharing your ideas. I agree with this too. And this is currently under making. Do you guys know where can I get some good favicon?

**Comment by purachina** [comment]
The ones in wikipedia maybe? We could simply resize them.

[Qt svg](https://upload.wikimedia.org/wikipedia/commons/0/0b/Qt_logo_2016.svg) is public domain and the [GTK+ one](https://upload.wikimedia.org/wikipedia/commons/4/42/GTK%2B_logo.svg) is C[C BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/).

**Comment by SaintFenix** [comment]
[Electron Icon](https://en.wikipedia.org/wiki/Electron_(software_framework)#/media/File:Electron_0.36.4_Icon.png) licensing seems to be MIT, and is listed at the link.

I found a few Java icons, but none of them were first party, it seems that Java's official icon is licensed, and requires permission which isn't a huge surprise since it's Oracle. closest thing I could find was here: [Java Icon](http://www.oracle.com/us/technologies/java/java-licensing-logo-guidelines-1908204.pdf) But I think this is just for showing that something has a java license, not for the logo itself. I'm not sure.

**Comment by BharatKalluri** [comment]
Shall I start working on this? Seems we need only a java icon, which we can figure out later.

**Comment by LewisVo** [comment]
@BharatKalluri you are always welcomed to contribute.

**Comment by SaintFenix** [comment]
ooooh, we could just steal theirs... :P 
We'd need to add a few, since this is a Linux list after all, but that's a good start.

Perhaps I'll go over and ask in their issue section, and see if they'll let me take some icons with attribution. If not, we'll do it all by hand.

**Comment by luongvo209** [comment]
You don't have to ask @SaintFenix, any part of their work is licensed CC 1.0
